src/nwb_datajoint/spikesorting/sortingview.py

```diff
@@ -140,9 +156,10 @@ def url(self, key, sortingview_sorting_id: str = None):
                                          include_curation=True)
         return url
 
-    def insert_manual_curation(self, key: dict, description=''):
-        """Based on information in key for an AutoCurationSorting, loads the curated sorting from sortingview,
-        saves it (with labels and the optional description, and inserts it to CuratedSorting
+    @staticmethod
+    def insert_manual_curation(key: dict, description=''):
```

Comment:
I see in line 612 of `AutomaticCuration` that the description for inserting automatically curated result to `Curation` is 'auto curated'. should the default description here then be `manual curated`?